app/Model/Attribute.php

```diff
@@ -3909,6 +3908,74 @@ public function editAttribute($attribute, $eventId, $user, $objectId, $log = fal
         return true;
     }
 
+    public function __delete($id, $user, $hard = false)
```

Comment:
public function, should be called "delete()"
private would be __delete()
protected would be _delete()

Reply:
However, delete on the model level is already defined, so consider a different name.

Reply:
Called it ``deleteAttribute``